src/components/Main.tsx

```diff
@@ -46,77 +46,61 @@ const Urls = {
     w2s: '/income/w2jobinfo',
     f1099s: '/income/f1099s'
   },
-  createPdf: '/createpdf'
+  createPdf: '/createpdf',
+  default: ''
 }
+Urls.default = Urls.taxPayer.info
 
 const drawerSections: Section[] = [
   {
     title: 'Personal',
     items: [
-      ['Taxpayer Information', Urls.taxPayer.info],
-      ['Spouse and Dependents', Urls.taxPayer.spouseAndDependent],
-      ['Filing Status', Urls.taxPayer.filingStatus],
-      ['Contact Information', Urls.taxPayer.contactInfo]
+      item('Taxpayer Information', Urls.taxPayer.info, <TaxPayerInfo />),
+      item('Spouse and Dependents', Urls.taxPayer.spouseAndDependent, <SpouseAndDependent />),
+      item('Filing Status', Urls.taxPayer.filingStatus, <FilingStatusSelect />),
+      item('Contact Information', Urls.taxPayer.contactInfo, <ContactInfo />)
     ]
   },
   {
     title: 'Income',
     items: [
-      ['Wages (W2)', Urls.income.w2s],
-      ['Income (1099)', Urls.income.f1099s]
+      item('Wages (W2)', Urls.income.w2s, <W2JobInfo />),
+      item('Income (1099)', Urls.income.f1099s, <F1099Info />)
     ]
   },
   {
     title: 'Results',
     items: [
-      ['Refund Information', Urls.refund],
-      ['Review and Print', Urls.createPdf]
+      item('Refund Information', Urls.refund, <RefundBankAccount />),
+      item('Review and Print', Urls.createPdf, <CreatePDF />)
     ]
   }
 ]
 
-const allUrls: string[] = (
-  drawerSections
-    .flatMap((section: Section) => section.items)
-    .map((item) => item[1])
-)
-
 export default function Main (): ReactElement {
-  const [, forward, prevUrl] = usePager(allUrls)
+  const allItems: SectionItem[] = drawerSections.flatMap((section: Section) => section.items)
+
+  const [, onAdvance, prev] = usePager(allItems, (item) => item.url)
```

Comment:
any reason for the leading comma in 
`const [, onAdvance, prev]`

Reply:
If you're asking about what that leading comma does, it's just selecting the middle two elements from the array returned by that usePager function. I thought the pager hook would return other info like the current page. But this is the only use of it so I simplified it in 9bbb11b.

Reply:
That makes sense, thanks for the context